packages/ui/src/primitives/icon.tsx

```diff
@@ -692,3 +692,14 @@ export const Checkmark = createIcon({
     />
   ),
 });
+
+export const PencilUnderlined = createIcon({
+  displayName: 'IconPencilUnderlined',
+  viewBox: '0 0 17 16',
```

Comment:
Not a dealbreaker but this caught my eye – is it intentional? Most of our icons tend to be `16 16` or `20 20`